Sources/Speechly/Speech/SpeechClient.swift

```diff
@@ -13,9 +13,10 @@ import NIO
 ///
 /// The client is ready to use once initialised.
 public class SpeechClient {
-    private let appId: UUID?
-    private let projectId: UUID?
-    private let appConfig: SluConfig
+    public let appId: UUID?
```

Comment:
Are these needed somewhere outside of the `SpeechClient`?

Reply:
Guess they can be made back into `private`, as they were only used in the demo app to determine if the client params have changed (basically to add support to setting `appId` in the Settings app on iOS). Probably that's not a need in the real world, or even if it is, it can be handled without exposing these.